content/source/docs/enterprise/before-installing/index.html.md

```diff
@@ -97,7 +97,7 @@ Terraform Enterprise currently supports running under the following operating sy
 
     - Debian 7.7+
     - Ubuntu 14.04.5 / 16.04 / 18.04 / 20.04
-    - Red Hat Enterprise Linux 7.4 - 7.8
+    - Red Hat Enterprise Linux 7.4 - 7.9
```

Comment:
Some existing update work is in flight for these here: https://github.com/hashicorp/terraform-website/pull/1645

Reply:
reverted